packages/design-tokens-schema/src/basis-tokens.ts

```diff
@@ -0,0 +1,173 @@
+  .string()
+  .trim()
+  .startsWith('{')
+  .endsWith('}')
+  .transform((value) => {
+    return value
+      .slice(1, -1) // remove { and }
+      .split('.'); // Get the individual parts
+  })
+  // Validate that each part is a valid identifier
+  .pipe(z.array(BaseDesignTokenIdentifierSchema))
+  // Join them back together
+  .transform((value) => `{${value.join('.')}}`);
+export type TokenRef = z.infer<typeof TokenRefSchema>;
+
+export const ColorWithRefSchema = BaseDesignTokenValueSchema.extend({
+  $type: z.literal('color'),
+  $value: TokenRefSchema,
+});
+export type ColorWithRef = z.infer<typeof ColorWithRefSchema>;
+
+export const ColorOrRefSchema = z.union([ColorTokenValidationSchema, ColorWithRefSchema]);
+export type ColorOrRef = z.infer<typeof ColorOrRefSchema>;
+
+export const ColorNameSchema = z.strictObject({
+  'bg-active': ColorOrRefSchema.optional(),
+  'bg-default': ColorOrRefSchema.optional(),
+  'bg-document': ColorOrRefSchema.optional(),
+  'bg-hover': ColorOrRefSchema.optional(),
+  'bg-subtle': ColorOrRefSchema.optional(),
+  'border-active': ColorOrRefSchema.optional(),
+  'border-default': ColorOrRefSchema.optional(),
+  'border-hover': ColorOrRefSchema.optional(),
+  'border-subtle': ColorOrRefSchema.optional(),
+  'color-active': ColorOrRefSchema.optional(),
+  'color-default': ColorOrRefSchema.optional(),
+  'color-document': ColorOrRefSchema.optional(),
+  'color-hover': ColorOrRefSchema.optional(),
+  'color-subtle': ColorOrRefSchema.optional(),
+});
+export type ColorName = z.infer<typeof ColorNameSchema>;
+
+export const BasisColorSchema = z.strictObject({
+  'accent-1': ColorNameSchema.optional(),
+  'accent-1-inverse': ColorNameSchema.optional(),
+  'accent-2': ColorNameSchema.optional(),
+  'accent-2-inverse': ColorNameSchema.optional(),
+  'accent-3': ColorNameSchema.optional(),
+  'accent-3-inverse': ColorNameSchema.optional(),
+  'action-1': ColorNameSchema.optional(),
+  'action-1-inverse': ColorNameSchema.optional(),
+  'action-2': ColorNameSchema.optional(),
+  'action-2-inverse': ColorNameSchema.optional(),
+  default: ColorNameSchema.optional(),
+  'default-inverse': ColorNameSchema.optional(),
+  disabled: ColorNameSchema.optional(),
+  'disabled-inverse': ColorNameSchema.optional(),
+  highlight: ColorNameSchema.optional(),
+  'highlight-inverse': ColorNameSchema.optional(),
+  info: ColorNameSchema.optional(),
+  'info-inverse': ColorNameSchema.optional(),
+  negative: ColorNameSchema.optional(),
+  'negative-inverse': ColorNameSchema.optional(),
+  positive: ColorNameSchema.optional(),
+  'positive-inverse': ColorNameSchema.optional(),
+  selected: ColorNameSchema.optional(),
+  'selected-inverse': ColorNameSchema.optional(),
+  transparent: ColorOrRefSchema.optional(),
+  warning: ColorNameSchema.optional(),
+  'warning-inverse': ColorNameSchema.optional(),
+});
+export type BasisColor = z.infer<typeof BasisColorSchema>;
+
+export const BasisTokensSchema = z.object({
```

Comment:
voor nu valideren we alleen kleuren, andere properties zijn toegestaan en worden ook in de parse data teruggegeven.